cfgov/paying_for_college/disclosures/scripts/api_utils.py

```diff
@@ -99,52 +192,19 @@
     'student.fafsa_sent.5plus_college_allyrs',
     'student.size',
     'student.enrollment.all',  # blank for many schools
-    'admissions.admission_rate.overall',
-    'admissions.admission_rate.by_ope_id',
     'student.retention_rate.four_year.full_time',
     'student.retention_rate.lt_four_year.full_time',
     'student.retention_rate.four_year.part_time',
     'student.retention_rate.lt_four_year.part_time',
     'student.demographics.veteran',  # blank for many schools
-    'aid.federal_loan_rate',
-    'aid.cumulative_debt.number',
-    'aid.cumulative_debt.90th_percentile',
-    'aid.cumulative_debt.75th_percentile',
-    'aid.cumulative_debt.25th_percentile',
-    'aid.cumulative_debt.10th_percentile',
-    'aid.median_debt_suppressed.overall',
-    'aid.median_debt_suppressed.completers.overall',
-    'aid.median_debt_suppressed.completers.monthly_payments',
-    'aid.students_with_any_loan',
-    'repayment.3_yr_repayment_suppressed.overall',
-    'repayment.repayment_cohort.1_year_declining_balance',
-    'repayment.1_yr_repayment.completers',
-    'repayment.1_yr_repayment.noncompleters',
-    'repayment.repayment_cohort.3_year_declining_balance',
-    'repayment.3_yr_repayment.completers',
-    'repayment.3_yr_repayment.noncompleters',
-    'repayment.repayment_cohort.5_year_declining_balance',
-    'repayment.5_yr_repayment.completers',
-    'repayment.5_yr_repayment.noncompleters',
-    'repayment.repayment_cohort.7_year_declining_balance',
-    'repayment.7_yr_repayment.completers',
-    'repayment.7_yr_repayment.noncompleters',
-    'earnings.6_yrs_after_entry.working_not_enrolled.mean_earnings',
-    'earnings.6_yrs_after_entry.median',
-    'earnings.6_yrs_after_entry.percent_greater_than_25000',
-    'earnings.7_yrs_after_entry.mean_earnings',
-    'earnings.7_yrs_after_entry.percent_greater_than_25000',
-    'earnings.8_yrs_after_entry.mean_earnings',
-    'earnings.8_yrs_after_entry.median_earnings',
-    'earnings.8_yrs_after_entry.percent_greater_than_25000',
-    'earnings.9_yrs_after_entry.mean_earnings',
-    'earnings.9_yrs_after_entry.percent_greater_than_25000',
-    'earnings.10_yrs_after_entry.working_not_enrolled.mean_earnings',
-    'earnings.10_yrs_after_entry.median',
-    'earnings.10_yrs_after_entry.percent_greater_than_25000'
 ]
 
 
+def api_school_query(school_id, fields):
```

Comment:
I see a test for this method but then it isn't referenced anywhere else in the code. Is it a placeholder for things to come?

Reply:
This has a useful method for exploring the scorecard API, which we've needed to do a lot lately since the Dept. of ED just unveiled a batch of potentially valuable endpoints. It has also helped identify endpoints that mostly deliver nulls, which is hard to plan for.